src/app.ts

```diff
@@ -830,27 +842,28 @@ export class App {
   }
 
   private writeResourceTests(scope: TypescriptTextWriter, api: gapi.client.discovery.RestDescription, ancestors: string, resourceName: string, resource: gapi.client.discovery.RestResource) {
-    for (const methodName in resource.methods) {
-      scope.comment(resource.methods[methodName].description);
+    _.forEach(resource.methods, (method, methodName) => {
+      scope.comment(method.description);
       scope.newLine(`await ${ancestors}.${resourceName}.${methodName}(`);
-      const params: Record<string, gapi.client.discovery.JsonSchema> = { ...resource['parameters'], ...resource.methods![methodName].parameters };
+      // tslint:disable-next-line:no-any
+      const params: Record<string, gapi.client.discovery.JsonSchema> = { ...(resource as any).parameters, ...resource.methods![methodName].parameters };
```

Comment:
I think we can just remove `...(resource as any).parameters,` to get rid of `any` here.

`resource` is clearly `gapi.client.discovery.RestResource` which doesn't have `.parameters`.

I've run test with `sheets` and no `resource` had `.parameters`.

Reply:
Yeah, I further tested this by adding:

```ts
if ('undefined' !== typeof (resource as any).parameters) {
  console.log('resource.parameters', (resource as any).parameters)
}
```

and generating _all_ of the types:

```
$ npm start | grep "resource.parameters" | wc -l
0
```

so I feel good about removing that.